### ISSUE TITLE
Plot history

### ISSUE THREAD

**Issue by tlnagy** [JunoLab/atom-julia-client#122]
It would nice to have simple history for plots in the plot pane (like RStudio).

**Comment by MikeInnes** [comment]
Definitely agreed, right now the plot pane is about the most simple thing it could be but these kinds of improvements are on the roadmap.

**Comment by ChrisRackauckas** [comment]
Agreed. I kinda think this should be pretty high on the roadmap, just below better console and progress meters.

**Comment by xgdgsc** [comment]
Any progress?

**Comment by pfitzseb** [comment]
Yes, we've had that for a couple of releases now.
